test/helpers/local_time_helper_test.rb

```diff
@@ -17,6 +17,11 @@ class LocalTimeHelperTest < TestCase
   include ActionView::Helpers::DateHelper, ActionView::Helpers::TagHelper
   include LocalTimeHelper
 
+  def normalize(html)
+    n = Nokogiri::HTML(html).xpath('//time').first
```

Comment:
Thanks guys, I left it as a quick hack and then forgot about it while doing the other changes.